scripts/dracula.sh

```diff
@@ -64,7 +65,11 @@ main()
   if $show_weather; then
     $current_dir/sleep_weather.sh $show_fahrenheit &
   fi
-
+  
+  # start cpu script in the background
```

Comment:
I don't think we need this code block, we only have it for sleep_weather.sh because that is a wrapper script that writes the data to data/weather.txt.

Reply:
I now realize that its a bit unclear what code block I'm referring to. I'm talking about the one starting with `# start cpu script in the background`

Reply:
Okay, I understand, I'll remove it, thanks for pointing it out, I wasn't sure if I was supposed to run it or not.